aggregate.go

```diff
@@ -74,7 +75,7 @@ func (a *Aggregator) Aggregate(bundles []Bundle) ([][]Bundle, error) {
 			continue
 		}
 
-		if distinctCount(bundles) != len(newGroup) {
+		if a.aggregatedBundlesMissVersionBundle(bundles, newGroup) {
```

Comment:
I am wrapping this for debugging purposes. This keeps the logic here nice and clean and gives us more control about all the details down the road.